src/app/zap-templates/zcl/zcl-with-test-extensions.json

```diff
@@ -297,7 +297,6 @@
             "RegisteredClients",
             "ICDCounter",
             "ClientsSupportedPerFabric",
-            "MaximumCheckInBackOff",
```

Comment:
Why this change?

Reply:
Alchemy did this because the attribute is named "MaximumCheckInBack**o**ff", not "MaximumCheckInBack**O**ff"

Reply:
Reverted changes to the zcl.json and zcl-with-test-extensions.json since the change is not related to the current cluster.